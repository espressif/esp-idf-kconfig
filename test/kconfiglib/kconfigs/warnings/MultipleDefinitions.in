mainmenu "Symbol name defined multiple times"

    config FEATURE
        bool "Feature"
        default y

    config FEATURE
        bool "Feature"
        default y
