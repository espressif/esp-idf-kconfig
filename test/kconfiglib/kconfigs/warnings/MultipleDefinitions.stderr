warning: Symbol FEATURE defined in multiple locations:
MultipleDefinitions.in:3
MultipleDefinitions.in:7
